randovania/data/json_data/prime3.txt

```diff
@@ -3575,7 +4427,7 @@ Asset id: 8007962190215742654
 Processing Access
 Asset id: 11685491285383767107
 > Dock to Scrapvault; Heals? False
-  * Normal Door to Scrapvault/Dock to Processing Access
+  * Plasma Door to Scrapvault/Dock to Processing Access
   > Room Center
       Trivial
 
```

Comment:
For the two Transit Stations below:

- 1-A
  - Add node "Tram to Pirate Research - Transit Station 1-B" and connect to door as Trivial. Make this a teleporter node.
  - Connect door to tram as trivial
- 1-B
  - Add node "Tram to Pirate Research - Transit Station 1-A" and connect to door as Trivial. Make this a teleporter node.
  - Connect door to tram as trivial

Reply:
Left these as generic nodes for now